combirepo/repository_combiner.py

```diff
@@ -874,11 +876,7 @@ def generate_mic_config(output_directory_path, temporary_directory_path):
     parser.set("create", "tmpdir", mic_directory_path)
     parser.set("create", "cachedir", mic_cache_directory_path)
     parser.set("create", "outdir", output_directory_path)
-    package_manager = None
-    if rpm_patcher.developer_disable_patching:
-        package_manager = "yum"
-    else:
-        package_manager = "zypp"
+    package_manager = "zypp"
```

Comment:
Why did you remove yum?
Some distros only have yum in repository.

Reply:
It's temporarily removed for debug purposes.
Anyway it's not clear why --disable-rpm-patching causes yum using.

Reply:
http://jira.rnd.samsung.ru/browse/TC-333

Reply:
> This issue can't be viewed

Reply:
@rayslava ?

Reply:
@ilya-palachev  fixed already

Reply:
> For now the default package manager is zypper, but it fails with strange errors ("uninstallable providers") when trying to install A sanitized and B non-sanitized when B depends on A. It seems to me that switching to yum will fix this.

If I recall correctly, there were strange problems with `zypper` and they were very hard to debug because of bindings. That's why I tried to use `yum` everywhere if possible.

But if the more modern version of `zypper` is better, you can use that instead. But it will require you some time to check that.

Reply:
As I see in combirepo git history zypp is used to be a default manager. It seems rather stable.

Reply:
I've no objections. Using zypp or yum is up to you.